.circleci/config.yml

```diff
@@ -1,27 +1,409 @@
-# This CircleCI config prevents CircleCI from running and creating failed
-# checks due to a missing or invalid CircleCI config on dev-preview
-# and branches based on dev-preview.
-#
-# By setting this job to only run on a non-existent branch, we prevent
-# execution from occuring on any branch with this config.
+# .circleci/config.yml
 
 version: 2.1
 
-# Define the jobs we want to run for this project
+parameters:
+  ios-destination:
+    type: string
+    default: platform=iOS Simulator,name=iPhone 14,OS=latest
+  macos-destination:
+    type: string
+    default: platform=macOS,arch=x86_64
+
+orbs:
+  # Using inline orb for now
+  getting-started-smoke-test:
+    orbs:
+      macos: circleci/macos@2
+      node: circleci/node@5.0.2
+      ruby: circleci/ruby@1.6.0
+    commands:
+      run-with-retry:
+        description: Run command with retry
+        parameters:
+          label:
+            description: Display name
+            type: string
+          command:
+            description: Command to run
+            type: string
+          retry-count:
+            description: Number of retry
+            type: integer
+            default: 3
+          sleep:
+            description: Wait duration until next retry
+            type: integer
+            default: 5
+          no_output_timeout:
+            description: Elapsed time the command can run without output
+            type: string
+            default: 10m
+        steps:
+          - run:
+              name: << parameters.label >>
+              command: |
+                retry() {
+                  MAX_RETRY=<< parameters.retry-count >>
+                  n=0
+                  until [ $n -ge $MAX_RETRY ]
+                  do
+                      << parameters.command >> && break
+                      n=$[$n+1]
+                      sleep << parameters.sleep >>
+                  done
+                  if [ $n -ge $MAX_RETRY ]; then
+                    echo "failed: ${@}" >&2
+                    exit 1
+                  fi
+                }
+                retry
+              no_output_timeout: << parameters.no_output_timeout >>
+    jobs:
+      ios:
+        parameters:
+          xcode-version:
+            type: string
+            default: 13.3.0
```

Comment:
We should be using 13.4 now. A slightly earlier version may result in compilation errors which do not happen with a newer compiler.